src/base-config/keyboard.json

```diff
@@ -71,7 +71,13 @@
         }
     ],
     "edit.splitSelIntoLines": [
-        "Ctrl-Alt-L"
+        {
+           "key": "Ctrl-Alt-L"
```

Comment:
@prafulVaishnav are you suggesting that it should not be changed for any platform or that it's changed for Linux while the other systems remain the same?